jitpack.yml

```diff
@@ -0,0 +1,7 @@
+before_install:
```

Comment:
Memory's bit rusty I must confess but, I'm not sure we need the jitpack.yml at all for this fork 🤔. Did you try without and was not successful @fluiddot ?

Reply:
Nope, I haven't, I'll try it without this file.

For making these changes I checked how we did it on other packages and some contain the `jitpack.yml` file and others not 🤷‍♂️ , here are some references:

Packages that **contain** `jitpack.yml` file:
- [`react-native-safe-area-context`](https://github.com/wordpress-mobile/react-native-safe-area-context/compare/master...wordpress-mobile:feat/jitpack#diff-1604193a5efbe3735c6ab8b44601ad6e42df52f6834bcc155650e3eef5b3fa94)
- [`react-native-gesture-handler`](https://github.com/wordpress-mobile/react-native-gesture-handler/compare/master...wordpress-mobile:b80e959908b383a26d6e35d992d6d529efad0b16#diff-1604193a5efbe3735c6ab8b44601ad6e42df52f6834bcc155650e3eef5b3fa94)

Packages that **don't contain** `jitpack.yml` file:
- [`react-native-video`](https://github.com/wordpress-mobile/react-native-video/compare/master...wordpress-mobile:enable-android-build-of-standalone-package)
- [`react-native-svg`](https://github.com/wordpress-mobile/react-native-svg/pull/9/files?file-filters%5B%5D=.flow&file-filters%5B%5D=.gif&file-filters%5B%5D=.gradle&file-filters%5B%5D=.h&file-filters%5B%5D=.idl&file-filters%5B%5D=.java&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.m&file-filters%5B%5D=.md&file-filters%5B%5D=.pbxproj&file-filters%5B%5D=.peg&file-filters%5B%5D=.png&file-filters%5B%5D=.podspec&file-filters%5B%5D=.prefs&file-filters%5B%5D=.snap&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&file-filters%5B%5D=.xml&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile)

Reply:
Ah, thanks for the context. Cool, let's see.

Looking back in history, I think I introduced `cd`ing into subfolders when the package to be build was nested inside the root folder (like, the `android` folder was 2 levels deep, [see example here from the old days](https://github.com/wordpress-mobile/gutenberg-mobile/pull/284/files#diff-1604193a5efbe3735c6ab8b44601ad6e42df52f6834bcc155650e3eef5b3fa94R3)). For "simple" package forks like the one here where the `android` folder is not deeply nested I think we don't custom build instructions in jitpack.yml.

Reply:
Oh ok, thanks for the clarification! I was a bit confused about what parts were required because each package has its own differences 😄 .

Reply:
>I created this GB-mobile PR to test if we can remove the jitpack.yml file.

I've just tested with the RN binaries generated in this [PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3550) and it's safe to remove the file. I'll update this branch and the GB PRs.